Bottom: 21163739eaacb74fcd943476ad6dcb2d14276602
Top:    08554c396ba9477871ae9ae4cecbb814f8648d26
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 13:32:14 +0000

Board outline polygon generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 0a86fa4..2e299a1 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1811,3 +1811,8 @@ debug_polygon (PolygonType *p)
 	break;
     }
 }
+
+POLYAREA *board_outline_poly ()
+{
+
+}
